addons/miscFixes/fauxLAMG/CfgWeapons.hpp

```diff
@@ -0,0 +1,32 @@
+class CfgWeapons {
+    class LMG_Mk200_F;
+    class GVAR(LAMG): LMG_Mk200_F {
+        author = "Potato";
+        displayName = "KAC Light Assault Machine Gun";
+        descriptionShort = "Light Machine Gun<br />Caliber: 5.56x45 mm";
+        baseWeapon = QGVAR(LAMG);
+        recoil = "CUP_SAW_recoil";
+        magazineWell[] = {"CBA_556x45_MINIMI"};
+        magazines[] = {
+            "200Rnd_556x45_Box_F",
+			"200Rnd_556x45_Box_Red_F",
+			"200Rnd_556x45_Box_Tracer_F",
+			"200Rnd_556x45_Box_Tracer_Red_F",
+
+			"CUP_200Rnd_TE4_Red_Tracer_556x45_M249",
+			"CUP_200Rnd_TE4_Yellow_Tracer_556x45_M249",
+			"CUP_200Rnd_TE4_Green_Tracer_556x45_M249",
+			"CUP_200Rnd_TE1_Red_Tracer_556x45_M249",
+			"CUP_200Rnd_TE4_Green_Tracer_556x45_M249_Pouch",
+			"CUP_200Rnd_TE4_Red_Tracer_556x45_M249_Pouch",
+			"CUP_200Rnd_TE4_Yellow_Tracer_556x45_M249_Pouch",
+			"CUP_200Rnd_TE1_Red_Tracer_556x45_M249_Pouch",
+			"CUP_100Rnd_TE4_Green_Tracer_556x45_M249",
+			"CUP_100Rnd_TE4_Red_Tracer_556x45_M249",
+			"CUP_100Rnd_TE4_Yellow_Tracer_556x45_M249",
+			"CUP_200Rnd_TE4_Green_Tracer_556x45_L110A1",
+			"CUP_200Rnd_TE4_Red_Tracer_556x45_L110A1",
+			"CUP_200Rnd_TE4_Yellow_Tracer_556x45_L110A1"
```

Comment:
These are already part of the `CBA_556x45_MINIMI` magwell. Is there a reason to add them again? And for the space between the vanilla and CUP magazines?